### PR TITLE
302 App env

### PR DESCRIPTION
**Pull request by BSick7**
This PR introduces AppEnvs type and endpoints.

### CODE REVIEW

File: app_envs.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	"net/http"
+	"path"
+)
+
+type AppEnvs struct {
+	Client *Client
+}
+
+func (e AppEnvs) Get(appName, envName string) (*types.AppEnv, error) {
+	res, err := e.Client.Do(http.MethodGet, path.Join("apps", appName, "envs", envName), nil, nil, nil)
```
Comment (ssickles) [inline app_envs.go:15]: One thing I realized when working on my current card is that apps can not be referenced uniquely by org_name and app_name. You either have to include the stack_name as a third argument or use the app_id for uniqueness.

**Review by ssickles — CHANGES_REQUESTED** [review]
Everything looks good from a coding perspective but there is an issue with the parameters to the api endpoint and being able to uniquely identify an application.

File: app_envs.go (hunk 1 above)
Comment (BSick7) [reply app_envs.go:15]: That's kinda weird. 
The endpoint for apps is `/orgs/:org_name/apps/:name` and it's still dependent on uniqueness within a single stack?

I think an app should be unique across orgs.
It's nice when the CLI can just say `nullstone deploy <app> <env>` -- you don't need to know the stack.

**Review by ssickles — APPROVED** [review]
LGTM!

File: app_envs.go (hunk 1 above)
Comment (BSick7) [reply app_envs.go]: Updated to use `appId` instead of `appName`.

**Pull request merged**